src/view/analytics/load.js

```diff
@@ -0,0 +1,15 @@
+import { getEventPayload } from '../utils';
+
+/**
+ * Event fired as soon as a prompt is loaded in the DOM.
+ *
+ * @param {Array} prompts Array of prompts loaded in the DOM.
+ */
+export const manageLoadEvents = prompts => {
+	prompts.forEach( prompt => {
+		const payload = getEventPayload( 'loaded', 'passive', prompt );
```

Comment:
@leogermani I wasn't sure what to put for the `action_type` event param. Does `passive` make sense, since this event will fire without any user interaction?

Reply:
Ideally this should be one of the values described in the docs here -> https://github.com/Automattic/newspack-plugin/tree/master/includes/data-events#prompt_interaction

Reply:
Ah, so it should be `donation`, `registration` or `newsletters_subscription` if the loaded prompt contains one of the corresponding block types, or empty if it doesn't?

Reply:
Resolved by later discussions/changes